assets/chat/js/chat.js

```diff
@@ -1092,16 +1095,20 @@ class Chat {
   }
 
   onCONNECTING(url) {
-    if (this.authenticated) {
-      MessageBuilder.status(
-        `Connecting as ${this.user.username} to ${Chat.extractHostname(
-          url
-        )} ...`
-      ).into(this);
+    MessageBuilder.status(
+      `Connecting to ${Chat.extractHostname(url)} ...`
+    ).into(this);
+  }
+
+  onME(data) {
+    this.setUser(data);
+    if (data && this.connectUser) {
```

Comment:
`connectUser` seems unnecessary to me. Is there a scenario where we wouldn't want to load settings and whispers, but would want to set the user? You make use of this param for on-stream chat, but on-stream chat is meant to be used as a browser source in OBS, which won't have a session anyway.